FlorinPosea/teme/3.5.react-rgb-color-game/src/App.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import ColorSquare from "./components/color-square/ColorSquare";
+import "./App.css";
+
+const DEFAULT_SQUARES = 6;
+var winningcolor = "";
+var msg = "";
+
+function App() {
+  const [newList, setNewList] = useState(true);
```

Comment:
State-ul asta nu este folosit. Tu faci un setNewList in useEffect la inceput, dar "newList" nu este folosit nicaieri altundeva. Deci se poate sterge.
![image](https://user-images.githubusercontent.com/45266485/201000672-705fa622-19b8-4ce0-a543-baef05bfb6ed.png)